engine/src/cython/initialize.cpp

```diff
@@ -73,17 +78,48 @@ void create_logger(std::string fileName, std::string loggingName, int ralId, std
 		file_sink->set_pattern(fmt::format("%Y-%m-%d %T.%e|{}|%^%l%$|%v", ralId));
 	}
 
+	// We want ALL levels of info to be registered. So using by default `trace` level
 	file_sink->set_level(spdlog::level::trace);
 	spdlog::sinks_init_list sink_list = { stdout_sink, file_sink };
 	auto logger = std::make_shared<spdlog::async_logger>(loggingName, sink_list, spdlog::thread_pool(), spdlog::async_overflow_policy::block);
-	logger->set_level(spdlog::level::trace);
+	
+	// By default, will contains all sort of logs (more logs detail)
+	std::string logger_level_wanted = "trace";
+	auto log_level_it = config_options.find("LOGGING_LEVEL");
+	if (log_level_it != config_options.end()){
+		logger_level_wanted = config_options["LOGGING_LEVEL"];
+	}
+
+	// level of logs
+	if (logger_level_wanted == "off") {
```

Comment:
This mapping of string to spdlog enum is now happening twice can you please turn this into a function we can reuse.

Reply:
Currently `logger_level` is just once but the `flush_level` is mapping twice. So I am updating this `flush_level` in a separate function then we can reuse it.